src/shogun2-core/shogun2-service/src/test/java/de/terrestris/shogun2/service/ImageServiceTest.java

```diff
@@ -0,0 +1,147 @@
+package de.terrestris.shogun2.service;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.when;
+
+import java.awt.image.BufferedImage;
+import java.io.ByteArrayOutputStream;
+import java.util.Arrays;
+
+import javax.imageio.ImageIO;
+
+import org.junit.Test;
+import org.springframework.mock.web.MockMultipartFile;
+
+import de.terrestris.shogun2.dao.ImageDao;
+import de.terrestris.shogun2.model.Image;
+
+public class ImageServiceTest extends AbstractExtDirectCrudServiceTest<Image, ImageDao<Image>, ImageService<Image, ImageDao<Image>>> {
+
+	/**
+	 *
+	 * @throws Exception
+	 */
+	public void setUpImplToTest() throws Exception {
+		implToTest = new Image();
+	}
+
+	@Override
+	protected ImageService<Image, ImageDao<Image>> getCrudService() {
+		return new ImageService<Image, ImageDao<Image>>();
+	}
+
+	@SuppressWarnings("unchecked")
+	@Override
+	protected Class<ImageDao<Image>> getDaoClass() {
+		return (Class<ImageDao<Image>>) new ImageDao<Image>().getClass();
+	}
+
+	@SuppressWarnings("static-access")
+	@Test
+	public void upload_asExpected() throws Exception {
+
+		Image persistedImage = null;
+
+		BufferedImage bimg = new BufferedImage(
+				500,500,BufferedImage.TYPE_3BYTE_BGR);
+		ByteArrayOutputStream baos = new ByteArrayOutputStream();
+		ImageIO.write( bimg, "jpg", baos );
+		byte[] imageByteArray = baos.toByteArray();
+
+		MockMultipartFile mockMultipartFile = new MockMultipartFile(
+				"fileData",
+				"fileName.jpg",
+				"image/jpeg",
+				imageByteArray);
+
+		byte[] thumbnail = crudService.scaleImage(
```

Comment:
The static `scaleImage` method should be accessed in a static way, i.e. `ImageService.scaleImage()`